tests/conftest.py

```diff
@@ -131,6 +131,12 @@ def pytest_addoption(parser):
     parser.addoption("--deep_clean", action="store_true", default=False,
                      help="Deep clean DUT before tests (remove old logs, cores, dumps)")
 
+    ###############################
+    #   test_route_perf options   #
+    ###############################
+    parser.addoption("--num_routes", action="store", default=10000, type=int,
```

Comment:
The `--num_routes` option has been defined in tests/vxlan/conftest.py. There is conflicting. Suggest to move definition of this `--num_routes` specific to route testing to `tests/route/conftest.py`.

Reply:
Thanks for this suggestion! I did move it to `tests/route/conftest.py` but the PR tests still fail. And I could not reproduce this problem locally. Wondering if you have any idea about this?

Reply:
It turns out that the vxlan tests define the option in __init__.py and it can be seen outside. Moved the part to a separate conftest.py file resolves the problem.